visualization/arcadia/README.md

```diff
@@ -24,10 +24,10 @@ Arcadia is visual analytics and BI software that runs natively within modern dat
 
 
 ### Replace dataset_name and dataset_detail from `pulse.json`
-- Use `replace_dataset_name.sh` script to replace dataset_name and dataset_detail in pulse.json
+- Use `replace-dataset-name.sh.sh` script to replace dataset_name and dataset_detail in pulse.json
```

Comment:
I think this is likely an oversight (.sh.sh)

Reply:
Ideed, fixed.